Library/Source/XMLHttpRequest.cpp

```diff
@@ -4,6 +4,7 @@
 
 namespace babylon
 {
+
```

Comment:
if you are keen on adding empty lines after namespace, we should add a corresponding extra line before the end bracket and we should do it everywhere. at some point, we should add a coding guideline to the project.

Reply:
.clang-format to the rescue! Let's not waste too much time with coding guidelines for now. I have a nice format derived from MS standard. I'll do a separate PR for that.

Reply:
Ok, sounds good, but let's not add extra diff lines until we do that?